django/thunderstore/repository/templates/repository/package_docs.html

```diff
@@ -11,8 +11,7 @@
     <div class="card-header">Package format requirements</div>
     <div class="card-body markdown-body">
         <p>
-            A valid package is a zip file that contains at least the following
-            files:
+            A valid package is a zip file that contains the following files:
```

Comment:
* Should this page mention which version of the package format it's documenting?
* Should there be change log / diff / some other way for describing the differences between the versions?
* Should it mention our plans for deprecating the old versions (I know it's a irrelevant in this case since the only change is an addition of a optional field, but generally speaking)?

Reply:
> Should this page mention which version of the package format it's documenting?

Not at this point since it's entirely internal

> Should there be change log / diff / some other way for describing the differences between the versions?

Yeah probably, once the format is no longer just an internally tracked field.

> Should it mention our plans for deprecating the old versions (I know it's a irrelevant in this case since the only change is an addition of a optional field, but generally speaking)?

Also yes once it's no longer internal 😄

Before we can make the info public, we'll need to go through all the packages to infer which format they are (this isn't done yet, the field just exists for new uploads). After that's done we can expose it on the API and that's when I'd worry about documenting each version.